Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [qTox_enhanced/qTox](https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/qtox_enhanced/-/qtox_app/horizontal-auto.svg)

zoff99 commented:
@mveljko78 
did you check those lines with `calloc` `memcpy` `free`?
those are the most dangerous for memory issues.

https://github.com/Zoxcore/qTox_enhanced/pull/9/files#diff-4f4f4e9890503ec712c193d2a55ccd7eaa6d71f4cee9ad96b1b0b059447db87fR527-R535

mveljko78 commented:
Yes, I did. looks good

On Tue, Jan 3, 2023 at 3:40 PM Zoff ***@***.***> wrote:

> @mveljko78 <https://github.com/mveljko78>
> did you check those lines with calloc memcpy free?
> those are the most dangerous for memory issues.
>
>
> https://github.com/Zoxcore/qTox_enhanced/pull/9/files#diff-4f4f4e9890503ec712c193d2a55ccd7eaa6d71f4cee9ad96b1b0b059447db87fR527-R535
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Zoxcore/qTox_enhanced/pull/9#issuecomment-1369844360>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACL35BV6NCJB2SEV7DYBHYDWQQ26ZANCNFSM6AAAAAATNQEU5Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (703 of 703 strings)

Translation: qTox_enhanced/qTox
Translate-URL: https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/ru/
